java/test/compatibility_test.sh

```diff
@@ -14,6 +14,7 @@ fi
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )" #get the absolute diretory of this script
 cd $DIR
+rm -rf *columnstore* #remove old mariadb-columnstore-api directories from prior tests
```

Comment:
I'm a little worried this a little over broad wildcard and might unintentionally nuke new directories that someone added either for new code or temporary work. Is it possible to come up with a less broad pattern for the delete?